pyart/aux_io/rainbow_wrl.py

```diff
@@ -280,16 +284,18 @@ def read_rainbow_wrl(
     else:
         start_range = 0.0
     _range["data"] = np.linspace(
-        start_range + r_res / 2.0, float(nbins - 1.0) * r_res + r_res / 2.0, nbins
+        start_range + r_res / 2.0, float(maxbin - 1.0) * r_res + r_res / 2.0, maxbin
```

Comment:
@HappyXiaoAnAn @zssherman Heads up! This needs a check that range resolution is same for all sweeps. If not we have to raise an error as this can't be represented with the CfRadial1 data model.

Reply:
Thanks for reminding! I would like to add some code to check the range resolution. Does this make sense?
```python
# range
if not single_slice:
    # all sweeps have to have the same range resolution
    for i in range(nslices):
        slice_info = rbf["volume"]["scan"]["slice"][i]
        if not (slice_info['rangestep'] == common_slice_info["rangestep"]):
            raise ValueError("range resolution changes between sweeps")
```

Reply:
That looks OK to me.

Reply:
@kmuehlbauer Good point! Thanks for the heads up!

Reply:
That makes sense!! Thank you @HappyXiaoAnAn